### PR TITLE
Add eelek's SSB data and buff certain mons

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
Yes Pichu power will go first. If I have to ill make it 999 priority if someone out priority's this

File: mods/essb/moves.js [hunk 1]
```diff
@@ -1259,6 +1259,8 @@ exports.BattleMovedex = {
 		pp: 10,
 		priority: 0,
 		flags: {contact: 1, protect: 1, mirror: 1},
+		willCrit: true,
```
Comment (DeathlyPlays) [inline mods/essb/moves.js:1262]: Use ``isCrit: true,`` instead

**Review by DeathlyPlays — CHANGES_REQUESTED** [review]

File: mods/essb/moves.js (hunk 1 above)
Comment (ghost) [reply mods/essb/moves.js:1262]: K

**Comment by DeathlyPlays** [comment]
Also test it on a fresh copy of Exiled

File: mods/essb/moves.js (hunk 1 above)
Comment (ghost) [reply mods/essb/moves.js:1262]: I used frost breath and that what it said

**Comment by ghost** [comment]
K

**Review by DeathlyPlays — APPROVED** [review]

**Pull request merged**